syntax = "proto3";
package rofi.messages;

import "rofiCmd.proto";
import "jointResp.proto";

message RofiResp {
    int32 rofiId = 1;
    RofiCmd.Type respType = 2;

    JointResp jointResp = 3;
}
